index.ts

```diff
@@ -0,0 +1,91 @@
+import "reflect-metadata";
+import { PrismaClient } from "./src/generated/prisma-client/index";
+import { buildSchema } from "type-graphql";
+import { 
+  JournalEntryCrudResolver,
+  UserCrudResolver,
+  relationResolvers
+} from "./src/generated/type-graphql";
+import express from "express";
+import { ApolloServer } from "apollo-server-express";
```

Comment:
apollo-server-express is the Apollo Server package for express, the most popular Node.js web framework. It enables you to attach a GraphQL server to an existing Express server. It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.